app/views/components/submit_button.scala.html

```diff
@@ -17,5 +17,5 @@
 @()(implicit messages: Messages)
 
 <div class="section">
-    <button id="submit" class="button">@messages("site.continue")</button>
+  <button id="submit" class="button">@messages("site.continue")</button>
```

Comment:
Don't need the `<div>` here

Reply:
Not removing it because the 'section' class changes padding/margin of the element.